modules/ROOT/pages/clustering/clustering-advanced/reconciler.adoc

```diff
@@ -0,0 +1,58 @@
+[role=enterprise-edition]
+[[cluster-reconciler]]
+= Reconciler
+:description: This section describes how changes to the DBMS are processed by each server.
+
+[[cluster-reconciler-introduction]]
+== Introduction
+
+Administrative operations in Neo4j, such as creating a database, do not happen synchronously.
+Instead we have a model where the changes are written to the `system` database, and record the desired state of the DBMS.
+Each server has an internal component called the reconciler, which observes the desired state and makes changes to the local server to match that state.
+For example, the `system` database may record that it wants the database `bar` running on `server-1`.
+When the reconciler on `server-1` sees that, it will start `bar`.
+
+Executing an operation and getting back a successful response means that the desire is safely committed to the `system` database, and will be processed by each member of the cluster at some point, assuming the server is healthy enough.
```

Comment:
```suggestion
Executing an operation and getting back a successful response means that the desired state is safely committed to the `system` database, and that it will be processed by each member of the cluster at some point (assuming the server is healthy enough).
```
`desire` as a verb or noun is _very_ strong https://dictionary.cambridge.org/dictionary/english/desire
The adjective is okay, but I recommend reviewing the whole text and checking if another adjective is possible. For example, how about _intended state_?

Reply:
We use `requestedHosting` in `SHOW SERVERS`, _intended_ is fine by me but _requested_ is an existing term, so probably preferable?

Reply:
Yeah, _requested_ is also fine!